README.md

```diff
@@ -49,47 +49,64 @@ $client = new Client(new Settings(AuthToken::generate('username', 'password', 'd
 And then we can make next operations _(each call will returns an object with server response data)_:
 
 ```php
-<?php /** @var \Avtocod\B2BApi\Client $client */
+<?php
+
+use \Avtocod\B2BApi\Client;
+use \Avtocod\B2BApi\Params\DevPingParams;
+use \Avtocod\B2BApi\Params\DevTokenParams;
+use \Avtocod\B2BApi\Params\UserParams;
+use \Avtocod\B2BApi\Params\UserBalanceParams;
+use \Avtocod\B2BApi\Params\UserReportTypesParams;
+use \Avtocod\B2BApi\Params\UserReportsParams;
+use \Avtocod\B2BApi\Params\UserReportParams;
+use \Avtocod\B2BApi\Params\UserReportMakeParams;
+use \Avtocod\B2BApi\Params\UserReportRefreshParams;
```

Comment:
As I can see - this classes uses just once, so, `use` usage is required here? Maybe use "inline" instead?

Reply:
Here and below, of course

Reply:
I think inline usage in examples is very heavy. Maybe not showing `use` in examples?

If we look at the Laravel [documentation](https://laravel.com/docs/8.x/eloquent), `use` sometimes used, sometimes not:

```php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Flight extends Model
{
    /**
     * The storage format of the model's date columns.
     *
     * @var string
     */
    protected $dateFormat = 'U';
}
```

```php
<?php

class Flight extends Model
{
    const CREATED_AT = 'creation_date';
    const UPDATED_AT = 'last_update';
}
```

Reply:
nope, use inline, please

Reply:
Okay, I'm using inline c5e64d255039475edcc38439374cf0cc03678a06